src/main/java/com/marklogic/kafka/connect/sink/MarkLogicSinkConfig.java

```diff
@@ -54,6 +55,7 @@
 		.define(DMSDK_TRANSFORM, Type.STRING, Importance.MEDIUM, "Name of a REST transform to use when writing documents")
 		.define(DMSDK_TRANSFORM_PARAMS, Type.STRING, Importance.MEDIUM, "Delimited set of transform names and values")
 		.define(DMSDK_TRANSFORM_PARAMS_DELIMITER, Type.STRING, ",", Importance.LOW, "Delimiter for transform parameter names and values; defaults to a comma")
+		.define(DOCUMENT_COLLECTIONS_ADD_TOPIC, Type.BOOLEAN, false,Importance.LOW, "Boolean that indicate if topics should be add to documents collections ")
```

Comment:
Couple tweaks - could be "Boolean that indicates" (add an "s"), but I'd just do "Indicates if". 

And then the last part needs a little tweaking - I think the full string should be "Indicates if the topic name should be added to the set of collections for a document"